packages/testcontainers/src/generic-container/abstract-started-container.ts

```diff
@@ -76,6 +76,10 @@ export class AbstractStartedContainer implements StartedTestContainer {
     return this.startedTestContainer.getIpAddress(networkName);
   }
 
+  public getWaitStrategy() {
```

Comment:
Is it possible to test this new behaviour without exposing the underlying wait strategy?